server/s3server.cc

```diff
@@ -751,7 +751,6 @@ int main(int argc, char **argv) {
   S3Daemonize s3daemon;
   set_fatal_handler_exit();
   s3daemon.daemonize();
-  s3daemon.register_signals();
```

Comment:
you can keep this commented of in #if 0...  since other code is still in place

Reply:
ok.